src/us/kbase/workspace/database/provenance/Contributor.java

```diff
@@ -0,0 +1,220 @@
+package us.kbase.workspace.database.provenance;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Represents a contributor, which may be a person or an organization.
+ */
+
+public class Contributor {
+
+        public enum ContributorType {
+                PERSON,
+                ORGANIZATION
+        }
```

Comment:
Should this go into a class on its own? It seemed a bit OTT for an enum with two values, but it's easy enough to change.

Reply:
If it's used by other classes probably, but otherwise seems fine here